src/components/Form/Collection/Collection.jsx

```diff
@@ -242,6 +242,22 @@ export default class Collection extends ValidationElement {
       )
     }
 
+    // Create the 'byline' which is used to separate items as well as the bottom toggle action
+    const bylineBottom = (item, index) => {
+      if (this.state.items.length < 2) {
+        return ''
+      }
+
+      return (
+        <div className="byline bottom">
+          <a href="javascript:;;" className="toggle" onClick={this.toggle.bind(this, index)}>
```

Comment:
100% on board with leaving it as-is.  Making all the linters and scanners happy has always been an adventure when it comes to anchor tags, doubly so with a lot of the apps now that do routing dynamically.  Then you have something like `<a href="#next-page">` and the scanners get made that `#next-page` doesn't exist.  😱 

I didn't know all that context about when it's considered a link versus an anchor, though - that's cool.  But in any case, I was just curious if that was like the current "right way" - and regardless, I think it's something I'll adopt.  😛